microbench/src/main/java/io/netty/util/concurrent/ScheduleFutureTaskBenchmark.java

```diff
@@ -85,4 +86,22 @@ public void run() {
             }
         }).syncUninterruptibly();
     }
+
+    @Benchmark
+    @Threads(1)
+    public Future<?> scheduleLotsOutsideLoop(final ThreadState threadState) {
+        for (int i = 1; i <= threadState.num; i++) {
+            threadState.eventLoop.schedule(NO_OP, i, TimeUnit.HOURS);
```

Comment:
save `threadState.eventLoop` in a field outside the tight loop

Reply:
What for? It's a field, not a getter.

Reply:
schedule will likely use memory barriers that will force to load such fields on each iteration (unless `static final`): isn't a big deal, but saving this will make the bench more clean.

Reply:
done!